bundle/manifests/loki-operator.clusterserviceversion.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.4.0
+    operators.operatorframework.io/builder: operator-sdk-unknown
```

Comment:
How did this happen? Is this okay?

Reply:
Hrrr... operator-sdk 1.5.0 is somehow destroying this.

Reply:
Not really a blocks, this is informational.